documentation/RLC-Swagger-quickstart.md

```diff
@@ -0,0 +1,298 @@
+
+    **After the first generation, you need to switch `generate-metadata: false`  as we have some manual changes in this file and don't want them get overwrite by generated ones.**
+
+    ---  
+  
+1. **Edit rush.json**  
+    As the libraries in this azure-sdk-for-js repository are managed by rush, you need to add an entry in rush.json under projects section for the first time to make sure it works. For example:
+
+    ```
+        {
+          "packageName": "@azure-rest/agrifood-farming",
+          "projectFolder": "sdk/agrifood/agrifood-farming-rest",
+          "versionPolicyName": "client"
+        },
+    ```
+
+    Here you also need to replace the `packageName`, `projectFolder` into your own services'.
+
+    ---  
+    **NOTE**
+
+    About the `versionPolicyName`, if the library you are working on is for data-plane, then it should be `client`, if the library you are working on is for control plane, then it should be `mgmt`.  
+
+    ---  
+
+1. **Run autorest to generate the SDK**  
+
+    Now you can run this command in swagger folder you just created.
+
+    ```shell
+    autorest --typescript ./README.md
+    ```
+
+    After this finishes, you will see the generated code in `${PROJECT_ROOT}/src` folder.  
+    After that, you can get a workable package, and run the following commands to get a artifact if you like.
+
+    ```shell
+    rush update
+    rush build -t <your-package-name>
+    cd <your-sdk-folder>
+    rushx pack
+    ```
+
+    But we still need to add some tests for it.
+
+# Improve README.md document
+
+A minimal README.md is generated by typescript emitter and you could improve it with more meaningful information.
+
+
+# How to write test for RLC
+
+In order to release it, we need to add some tests for it to make sure we are delivering high quality packages. But before we add the test, we need to add a `generate-test: true` make the codegen generate the necessary change in `package.json` and `tsconfig.json` so that test framework can work. Once the generation finished, you will see a `sampleTest.spec.ts` file in your `{PROJECT_ROOT}/test/public` folder, which only has an empty test and you may change them into test against your own services.
+
+See the [Javascript Codegen Quick Start for Test](./Quickstart-on-how-to-write-tests.md) for information on how to write and run tests for the Javascript SDK.
+
+1. **Prerequisites**  
+
+    To record and playback the tests, [Docker](https://www.docker.com/) is required when we run the test, as the [test proxy server](https://github.com/Azure/azure-sdk-tools/tree/main/tools/test-proxy) is run in a container during testing. When running the tests, ensure the Docker daemon is running and you have permission to use it.
+
+1. **Write the test**
+    You could follow [the link](./Test-quickstart.md#example-1-basic-rlc-test-interaction-and-recording-for-azure-data-plane-service) to write your test step by step.
```

Comment:
Please update the reference here. Maybe there're references in other files not just in this PR